engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -287,7 +287,15 @@ public void copyInitialState(Path origAppDir) throws IOException
     FileStatus[] lFiles = fs.listStatus(origAppDir);
     for (FileStatus f : lFiles) {
       if (f.isDirectory()) {
-        String targetPath = f.getPath().toString().replace(origAppDir.toString(), newAppDir);
+
+        // In case of MapR/MapR-FS, f.getPath().toString() returns path as maprfs:///<orig app dir>
+        // whereas origAppDir.toString & newAppDir are in maprfs:/<orig or new app dir> format
+        // e.g.
+        // f.getPath().toString -> maprfs:///user/dtadmin/datatorrent/apps/application_1481890072066_0004/checkpoints
```

Comment:
Why is it in case of FileStatus Path toString have maprfs:/// whereas for origAppDir Path toString just have maprfs:/?